### PR TITLE
Environment variables updates for pointid-api & notification-push-api

### PR DESCRIPTION
**Pull request by ssylver93**

### CODE REVIEW

**Review by vivid-cpreston — APPROVED** [review]
Looks good on the terraform and github actions side - that said, I'm not as familiar with Spring, so you'll want someone else's confirmation for those bits

**Review by dhlevi — APPROVED** [review]

**Comment by dhlevi** [comment]
@ssylver93 Some cleanup on the conflicts, but that should be fairly minor to polish. We likely aren't using those files at all anymore anyway

**Review by yzlucas — APPROVED** [review]

File: server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties
Comment (vivid-cpreston) [inline server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties:1]: Looks like there's an unmerged change that needs fixing here!

File: server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application.properties
Comment (vivid-cpreston) [inline server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application.properties:1]: Checking ecs.tf, I see that the POINTID_ prefix is not used when passing in the environment variables. We need to either update ecs.tf to match this, or revert the addition of POINTID_ to the variable names.

The typo fix for the wfArcGis layer is a good catch!

**Review by vivid-cpreston — CHANGES_REQUESTED** [review]

File: server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Point ID secrets
 weather.user=#{WEATHER_USER}
 weather.password=#{WEATHER_PASSWORD}
+<<<<<<< HEAD
```
Comment (dhlevi) [inline server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties:4]: It looks like the <<<<< separators from Git are in the file. We want to remove these and make sure we don't have any duplicated content. These indicate conflicts and shouldn't be merged as changes

Comment (dhlevi) [reply server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application-secrets.properties:4]:
```
<<<<<<< HEAD
webade.oauth2.client.secret=#{POINTID_WEBADE_OAUTH2_CLIENT_SECRET}
=======
webade.oauth2.client.secret=#{WEBADE_OAUTH2_CLIENT_SECRET}
>>>>>>> d3f08d482eaafd2926976e2fa9258aa7ad988cc9
```

should be 

```
webade.oauth2.client.secret=#{WEBADE_OAUTH2_CLIENT_SECRET}
```

I think?

**Review by dhlevi — CHANGES_REQUESTED** [review]

**Review by dhlevi — APPROVED** [review]

**Comment by dhlevi** [comment]
@ssylver93 Changes are looking good. @vivid-cpreston Approve and merge once you're happy

File: server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application.properties [hunk 2]
```diff
@@ -18,7 +18,7 @@ max.allowed.radius=#{MAX_ALLOWED_RADIUS}
 async.job.interval=#{ASYNC_JOB_INTERVAL}
 async.job.ref.lat=#{ASYNC_JOB_REF_LAT}
 async.job.ref.lon=#{ASYNC_JOB_REF_LONG}
-async.job.ref.radius=#{ASYNC_JOB_REF_RADIUS}
+async.job.ref.radius=#{ASYNC_JOB_REF_LONG}
```
Comment (vivid-cpreston) [inline server/wfss-pointid-api/wfss-pointid-api-rest-endpoints/src/main/resources/application.properties:21]: This should be the radius, not the long - other than that looks good to me!

**Pull request merged**